### PR TITLE
Add a little newsletter utility

### PR DESCRIPTION
**Pull request by sethetter**
This is a little utility for quickly getting started on a new newsletter. It pulls data from Meetup, the Jobs form, and the Volunteering form, and puts it into a markdown file in the `content/newsletters/` folder.

### CODE REVIEW

File: newsletter/newsletter/.env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+EVENTS_URL=https://api.meetup.com/2/events?&sign=true&photo-host=public&group_urlname=devICT&limited_events=false&fields=series&status=upcoming&page=20
```
[the remark this answers is not in the document]
Comment (sethetter) [reply newsletter/newsletter/.env.example]: Yeah I don't know how that happened, lol.

**Comment by sethetter** [comment]
@jcbwlkr thanks for all the feedback!

**Comment by sethetter** [comment]
The zero value thing makes sense, I knew to do that, but forgot that the zero value of a slice is `nil`, not an empty slice. Thanks :D

**Comment by sethetter** [comment]
I wonder if there's a better way to handle recurring events? Maybe Listing a link to the next, but then the little blurb about it repeat underneath as a sub-bullet?

**Comment by jcbwlkr** [comment]
Screen Shot 2019-10-16 at 3 30 41 PM

For `bot events` we just put a disclaimer on there that said "recurring events only shown once". Maybe something simple like that here? I don't have a strong opinion

**Comment by sethetter** [comment]
Meetup returns a blurb about it's recurrence schedule, I just included that with the events

**Comment by jcbwlkr** [comment]
LGTM

**Pull request merged**